microscopium/preprocess.py

```diff
@@ -549,39 +557,58 @@ def find_background_illumination(fns, radius=51, quantile=0.05,
     --------
     `correct_image_illumination`, `correct_multiimage_illumination`.
     """
-    # This function follows the "PyToolz" streaming data model to
-    # obtain the illumination estimate. First, define each processing
-    # step:
-    read = io.imread
+    read = tlz.partial(io.imread)
```

Comment:
I have actually since seen that the toolz community uses tz, not tlz. I don't want you to clutter this PR with a huge rename/refactor, but maybe add an extra import, "import cytoolz as tz", and use that here?